isodesign/process/calculation.py

```diff
@@ -1,43 +1,96 @@
+""" Module for calculation """
+
 from itertools import product
+import pandas as pd
+
 
 class Tracer:
-    def __init__(self, name_mol, labeling):
-        self.name_mol = name_mol
+    """ 
+    This class is for the initiation of tracers.
+    """
+
+    def __init__(self, name, labeling):
+        self.name = name
         self.labeling = labeling
         self.num_carbon = None
 
     def __len__(self):
+        """ 
+        This method permit to count the carbon number by 
+        using the labeling.
+        """
         self.num_carbon = len(self.labeling)
 
     def __repr__(self) -> str:
-        return f"Molecule name: {self.name_mol}, Number of associated carbon(s) : {self.num_carbon}"
+        return f"Molecule name: {self.name}, Number of associated carbon(s) : {self.num_carbon}"
 
 
-def intra_mol_comb(num_mol, start, end, step):
-    fractions = []
+def intra_mol_comb(num_mol, step=10, start=0, end=100):
+    """
+    Function that output a list of combination for a molecule family.
+    Input : number of molecule in the family, step (10 by default), start (0 by default) 
+    and (100 by default)
```

Comment:
Pour les [docstrings](https://www.dataquest.io/blog/documenting-in-python-with-docstrings/), il y a plusieurs méthodes pour standardiser les descriptions des arguments. Celle que j'utilise personnellement est celle qui est donné par le formalisme de [RestructuredText](https://docutils.sourceforge.io/rst.html). Tu devrais y jeter un coup d'oeil et voir comment font les gens pour documenter leurs fonction/classes/modules! 

PS: dans mon code tu pourras trouver des exemples